file/lang/de

```diff
@@ -1,6 +1,6 @@
 index_title=Java-Dateimanager
 index_nojava=Dieses Modul benötigt Java, um zu funktionieren, aber Ihr Browser unterstützt Java nicht.
-index_eremote=Es gibt keine:n Unix-Benutzer:in, der dem Webmin-Login $1 entspricht.
+index_eremote=Es gibt keine:n Unix-Benutzer:in, der/die dem Webmin-Login $1 entspricht.
```

Comment:
Thanks! Though, isn’t there a more inclusive way to phrase this? What about using `xier` instead of the cumbersome `der/die`?

Reply:
This is the common and established way it is being used in German language.

Reply:
Alright!